script/install/debootstrap.bash

```diff
@@ -173,5 +173,49 @@ echo "$chroot_cmd update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9
 chroot "$ISOLATE_ROOT" update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 75
 # gcc 9 done
 
+[ -z "$TRAVIS" ] && bash script/confirm.bash 'Install .Net Core (C#)' && {
+  set -x
+
+  : Installing .Net Core for C#
+  # https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1604
+
+  : Downloading package info from Microsoft
+  UBUNTU_VERSION=$(lsb_release --release --short)
+  wget -q "https://packages.microsoft.com/config/ubuntu/${UBUNTU_VERSION}/packages-microsoft-prod.deb" -O packages-microsoft-prod.deb
+  dpkg -i packages-microsoft-prod.deb
+  rm packages-microsoft-prod.deb
+  ["$UBUNTU_VERSION" == "18.04"] && sudo add-apt-repository universe
+  apt-get install -y apt-transport-https
+  apt-get update
+
+  : Installing dotnet 3.1
+  apt-get install -y dotnet-sdk-3.1
```

Comment:
These will need to be done inside chroot.

Reply:
Whoops. Completely forgot about that. Will fix it up